docs/cni/kubernetes/simple-policy-demo/README.md

```diff
@@ -0,0 +1,131 @@
+# Kubernetes NetworkPolicy: A simple example 
+This guide provides a simple way to try out Kubernetes NetworkPolicy with Calico.  It requires a Kubernetes cluster configured with Calico networking, and expects that you have `kubectl` configured to interact with the cluster.
+
+You can quickly and easily deploy such a cluster by following one of the [getting started guides](../README.md#getting-started-guides)
+
+### Configure Namespaces
+This guide will deploy pods in a Kubernetes Namespaces.  Let's create the `Namespace` object for this guide.
+
+```
+# Create Namespace policy-demo
```

Comment:
This reads like "Run `Create Namespace policy-demo` as root, expecting output `kubectl create ns policy-demo`.

I'd don't think anyone will actually get this wrong, but it's not perhaps the perfect representation.